tests/console/firewalld.pm

```diff
@@ -41,7 +41,7 @@ sub test2 {
     assert_script_run("iptables -L IN_public_allow --line-numbers | sed '/^num\\|^\$\\|^Chain/d' | wc -l > /tmp/nr_rules.txt");
 
     # Check if it's tumbleweed or Leap/SLE and run the correct test accordingly
-    if (is_tumbleweed || is_sle('15-SP3+')) {
+    if (is_tumbleweed || is_sle('15-SP3+') || is_leap('15.3+')) {
```

Comment:
Unfortunately this was already done inefficiently in before.

Our tests should always consider openSUSE Tumbleweed as the default product without a version. That means any differing behaviour should have an explicit exclude rule for the "older products" and potentially an explanation why the behaviour should differ. You can try to negate the logic check

```suggestion
    if (!is_sle('<15-SP3') && !is_leap('<15.3')) {
```

@foursixnine you introduced the "15-SP3+" in https://github.com/michelmno/os-autoinst-distri-opensuse/commit/7152ed2f85e49676f730d0d25c1cfa39de02ee32 and @atanno you introduced the "is_tumbleweed" in https://github.com/michelmno/os-autoinst-distri-opensuse/commit/0e5f7e6a2e91f66145ff782276d488f0f5065146